### ISSUE TITLE
Translations and the i tag

### ISSUE THREAD

**Issue by eduardopato41** [p4535992/inventory-plus#5]
Hi again! Sorry for bothering you with this, but it seems the translation is not working right now. I don't know if its because it is not included in the module.json under languages or something else. For my own use, I added my translation in the /lang/en.json but I wanted to let you know.

I also added the new strings in the file (will paste the complete translation in the end of the post).

One more thing, the "+ Add Custom Category" is loading with the font-family of the plus icon, because your are closing the `<i>` tag after the text. I edited to be like this and its all right again:
```
<a class="custom-category"><i class="fas fa-plus"></i> ${i18n(`${CONSTANTS.MODULE_NAME}.inv-plus-dialog.addcustomcategory`)}</a>
```

Thanks for all your work!

New pt-br translation:
```
{
  "inventory-plus.setting.reset.name": "Redefinir as configurações para o padrão",
  "inventory-plus.setting.reset.label": "Redefinir as Configurações",
  "inventory-plus.setting.reset.hint": "Isso irá redefinir todas as configurações de volta para o padrão do sistema ativo.",
  "inventory-plus.setting.debug.name": "Ativar depuração",
  "inventory-plus.setting.debug.label": "Mostra mensagens de depuração no console",
  "inventory-plus.setting.debug.hint": " ",

  "inventory-plus.setting.enableItemTransfer.name": "Recurso: Ativar transferência de itens",
  "inventory-plus.setting.enableItemTransfer.hint": "Mesmo que seja o recurso principal deste módulo, você ainda pode desativar a transferência de itens. Saiba que você pode desativar temporariamente o recurso de transferência pressionando a tecla ALT enquanto move o item.",
  "inventory-plus.setting.enableCurrencyTransfer.name": "[Somente com 'Recurso: Ativar transferência de itens' ativado] Ativar transferência de moeda",
  "inventory-plus.setting.enableCurrencyTransfer.hint": "Ao mover um objeto chamado \"Moeda\" será aberta uma caixa de diálogo para transferir dinheiro em vez de transferir o item real.",
  "inventory-plus.setting.actorTransferSame.name": "[Somente com 'Recurso: Ativar transferência de item' ativado] Ativar para atores do mesmo tipo",
  "inventory-plus.setting.actorTransferSame.hint": "Se estiver desativado, você terá que confiar nos pares que você pode definir manualmente.",
  "inventory-plus.setting.actorTransferPairs.name": "[Somente com 'Recurso: Ativar transferência de itens' ativado] Tipos de atores compatíveis:",
  "inventory-plus.setting.actorTransferPairs.hint": "O corpo de um mapa JSON de pares de valor chave de tipo de atores compatíveis que permitem a transferência. Por padrão, este módulo só funciona entre folhas de atores do mesmo tipo, mas alguns sistemas podem ter outras combinações que funcionam. Exemplo 1: \"character\":\"synthetic\",\"synthetic\":\"vehicles\",\"vehicles\":\"character\". Exemplo 2 (para DnD5e): \"character\":\"vehicle\",\"vehicle\":\"character\" que permitiria mover itens entre personagens e veículos. Você também pode ter vários tipos de ator como valor, exemplo 3: \"character\":[\"vehicle\", \"character\"],\"vehicle\":\"character\" permitiria a transferência de personagem para veículo, de personagem para personagem e de veículo para personagem, mas NÃO de veículo para veículo (desde que a caixa de seleção \"atores do mesmo tipo\" esteja desmarcada).",

  "inventory-plus.setting.enableInventorySorter.name": "Recurso: Ativar ordenador de inventário",
  "inventory-plus.setting.enableInventorySorter.hint": "Ordena automaticamente todos os itens dos atores (inventário, recursos e magias) em ordem alfabética (dentro de cada categoria).",

  "inventory-plus.setting.enableEquipmentMultiplier.name": "Recurso: Multiplicador de Equipamento",
  "inventory-plus.setting.enableEquipmentMultiplier.hint": "Aplique um multiplicador de peso aos itens equipados",
  "inventory-plus.setting.equipmentMultiplier.name": "[Somente com 'Recurso: Multiplicador de Equipamento' ativado] Valor do Multiplicador de Equipamento",
  "inventory-plus.setting.equipmentMultiplier.hint": "Qual multiplicador de peso aplicar aos itens equipados",

  "inventory-plus.setting.enableIntegrationWithVariantEncumbrance.name": "Sobrecarga Variante",
  "inventory-plus.setting.enableIntegrationWithVariantEncumbrance.hint": "Se habilitado, calculará o peso com a API do módulo 'VariantEncumbrance' para um cálculo melhor e mais inteligente do peso, caso o módulo esteja presente e ativo",

  "inventory-plus.dialogs.resetsettings.title": "Redefinir Configurações do Módulo",
  "inventory-plus.dialogs.resetsettings.content": "Tem certeza de que deseja redefinir todas as configurações do módulo para os padrões do sistema atual? <strong>ISTO NÃO PODE SER DESFEITO!</strong>",
  "inventory-plus.dialogs.resetsettings.confirm": "Redefinir",
  "inventory-plus.dialogs.resetsettings.cancel": "Cancelar",

  "inventory-plus.inv-plus-dialog.categoryname": "Nome da Categoria:",
  "inventory-plus.inv-plus-dialog.maxweightdescription": "Impede que qualquer item seja colocado na categoria se o peso do item exceda o limite desse número.",
  "inventory-plus.inv-plus-dialog.maxweightprefix": "Peso Máximo:",
  "inventory-plus.inv-plus-dialog.inherentweightdescription": "Se definido para qualquer número acima de 0, o peso será aumentado nesse valor em sua Capacidade de Carga.",
  "inventory-plus.inv-plus-dialog.inherentweightprefix": "Peso da Categoria:",
  "inventory-plus.inv-plus-dialog.ignoreweightdescription": "Se marcado, o peso dos itens nesta categoria não serão calculados em sua Capacidade de Carga.",
  "inventory-plus.inv-plus-dialog.ignoreweightprefix": "Ignorar Peso dos Itens:",

  "inventory-plus.inv-plus-dialog.addcustomcategory": "Adicionar Categoria Personalizada",
  "inventory-plus.inv-plus-dialog.creatingnewinventorycategory": "Criando Nova Categoria de Inventário",
  "inventory-plus.inv-plus-dialog.accept": "Aceitar",
  "inventory-plus.inv-plus-dialog.cancel": "Cancelar",
  "inventory-plus.inv-plus-dialog.deletecategoryprefix": "Excluir",
  "inventory-plus.inv-plus-dialog.deletecategory": "Excluir Categoria",

  "inventory-plus.dialogs.warn.notargethtml": "Nenhum html de destino encontrado para a queda do item. Geralmente acontece quando a queda ocorre em um beco sem saída, tentar novamente arrastar e soltar é a solução mais simples",
  "inventory-plus.dialogs.warn.nocategoryfounded": "Nenhuma categoria fundada com o tipo '{targetType}'",
  "inventory-plus.dialogs.warn.itemtypecurrent": "O tipo não é 'Item'",
  "inventory-plus.dialogs.warn.itemid": "Nenhum itemId encontrado para o item",
  "inventory-plus.dialogs.warn.itemcurrent": "Nenhum itemCurrent encontrado para o item",
  "inventory-plus.dialogs.warn.itemdata": "Nenhum itemData encontrado para o item",
  "inventory-plus.dialogs.warn.exceedsmaxweight": "O item não pode ser inserido porque excede o peso máximo na categoria '{categoryName}'",
  "inventory-plus.dialogs.warn.nodroppeditem": "Nenhum item encontrado",

  "inventory-plus.notEnoughCurrency": "Você não tem o suficiente das seguintes moedas",
  "inventory-plus.howManyItem": "Quantos itens você deseja mover?",
  "inventory-plus.one": "Um",
  "inventory-plus.half": "Metade",
  "inventory-plus.max": "Máximo",
  "inventory-plus.stackItems": "Juntar itens do mesmo tipo",
  "inventory-plus.transfer": "Transferir",
  "inventory-plus.howMuchCurrency": "Quanto você quer mover?",
  "inventory-plus.currency": "Moeda",
  "inventory-plushowmanytimeswanttomove": "Quantos itens você deseja mover?"
}
```

**Comment by p4535992** [comment]
sorry to many modules to follow... now it should be fine...i think ?

**Comment by p4535992** [comment]
for the font bug you must wait for the next release

**Comment by p4535992** [comment]
Solved with 0.5.9
